CNAME

```diff
@@ -1 +1 @@
-gci18.fossasia.org
```

Comment:
Why these changes?  Revert it.

Reply:
It's already reverted and the CNAME content is brought back. I accidentally changed it while testing my changes.